packages/shared/routes/dashboard/wallet/views/AccountNavigation.svelte

```diff
@@ -65,7 +56,7 @@
 </style>
 
 <div class="flex flex-row justify-between items-start py-5" bind:this={rootElement}>
-    <button data-label="back-button" class="flex-1 mt-1" on:click={handleBackClick} bind:this={buttonElement}>
+    <button data-label="back-button" class="flex-1 mt-1" on:click={() => {}} bind:this={buttonElement}>
```

Comment:
Probably should just remove this button all together?

Reply:
Didnt want to remove it because it will break the layout because the parent has `flex justify-between`. So I thought of making the right changes on the next PR that is the new top nav